README.md

```diff
@@ -222,29 +230,23 @@ We recommend that you copy them into your own app and modify them to suit your n
 
 ### What this project is doing accessibility-wise?
 
-This project manages two types of Accordions, with single or multiple items open.
+This project manages Accordions with several options available for allowing/not allowing multiple items to be open at once, and allowing/not allowing all items to be closed.
```

Comment:
I'm not convinced that the distinction between these two content structures can be described by Accordions and Disclosures. The spec states:

> If the implementation allows only one panel to be expanded

The language - while not explicit - does imply that multiple items _may_ be expanded, and that it's up to the implementation. The distinction between an 'Accordion' and 'Disclosure' is not very explicit, but it would appear that it really only has to do with logical grouping of related items.

Reply:
Ok, fair point. I read further into the example page and now agree with you, accordion is the umbrella term and we need to simply define the attributes given each different option...not much changes really (in terms of attributes) 🎉